modules/framework/galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/routes/RunQueryRoute.java

```diff
@@ -392,8 +386,8 @@ private List<IRunResult> getRunsByRunName(String runName) throws ResultArchiveSt
 		return runs;
 	}
 
-    private List<RasRunResult> convertRunsToRunResults(List<IRunResult> runs,boolean isMethodDetailsRequested) throws ResultArchiveStoreException {
-		boolean isShort = isTestRunShort(isMethodDetailsRequested);
+    private List<RasRunResult> convertRunsToRunResults(List<IRunResult> runs) throws ResultArchiveStoreException {
+		boolean isShort = isTestRunShort(rasDetailsQueryParams.getDetail());
```

Comment:
isShort and isTestRunShort could confuse people - perhaps this should be something like isMethodDetailsExcluded? Or something better?